state/storage.go

```diff
@@ -756,7 +749,7 @@ func (s *Storage) LatestEventsInRooms(userID string, roomIDs []string, to int64,
 // visibleEventNIDsBetweenForRooms determines which events a given user has permission to see.
 // It accepts a nid range [from, to]. For each given room, it calculates the NID ranges
 // [A1, B1], [A2, B2], ... within [from, to] in which the user has permission to see events.
```

Comment:
Can we update this to say that we only return one NID range, please?

Reply:
Bah, forgot that comment :D